### ISSUE TITLE
Keycloak DCR - M1

### ISSUE THREAD

**Issue by sedkis** [TykTechnologies/tyk-demo#167]
Anyone on m1 trying to run Keycloak DCR deployment will have to build Docker image locally.

Here are instructions:
https://github.com/docker/for-mac/issues/5310

**Comment by carlostyk** [comment]
Amazing! This was next on my "todo" list. Thank you

**Comment by sedkis** [comment]
Yeah, I'm not sure if we can automate this part.. building a docker image locally? perhaps we can actually. hmmm

**Comment by carlostyk** [comment]
I was able to make it work with this other image - https://hub.docker.com/r/dak0rn/keycloak-m1
